src/class-parsely.php

```diff
@@ -131,17 +132,32 @@ class Parsely {
 	 *
 	 * @since 3.9.0
 	 * @access private
+	 *
 	 * @var bool
 	 */
 	public $are_credentials_managed;
 
+	/**
+	 * Holds the managed options and their values.
+	 *
+	 * This allows hosting providers to provide a more customized experience for
+	 * the plugin by handling options automatically.
+	 *
+	 * @since 3.9.0
+	 * @access private
+	 *
+	 * @var array<empty>|array<string, bool|string|null>
```

Comment:
I think we can remove `array<empty>`

Reply:
PHPStan complains if we do:

 ------ ------------------------------------------------------------------------------
  Line   src/class-parsely.php
 ------ ------------------------------------------------------------------------------
  712    Property Parsely\Parsely::$managed_options (array<string, bool|string|null>)
         does not accept array<array<string>|bool|string|null>.
 ------ ------------------------------------------------------------------------------